src/components/MessageInput/__tests__/AttachmentPreviewList.test.js

```diff
@@ -17,6 +16,13 @@ const uploadsReducer = (pv, cv) => {
   return pv;
 };
 
+/**
+ * @param {string} word
+ * @returns string
+ */
+const capitalize = ([firstLetter, ...restOfTheWord]) =>
+  `${firstLetter.toUpperCase()}${restOfTheWord.join('')}`;
```

Comment:
@arnautov-anton this implementation does not follow the lodash definition:

"Converts the first character of string to upper case and the remaining to lower case."

[lodash/capitalize](https://lodash.com/docs/4.17.15#capitalize)

If the `restOfTheWOrd` contains uppercase letters, then those would be kept.

Reply:
This is just a utility function that has to capitalize the first letter of the word just the way it's designed, it doesn't need (nor is meant) to follow full `lodash/capitalize` implementation.